test/RSocketClientServerTest.cpp

```diff
@@ -36,15 +37,16 @@ TEST(RSocketClientServer, ConnectManySync) {
 TEST(RSocketClientServer, ConnectManyAsync) {
   auto server = makeServer(std::make_shared<HelloStreamRequestHandler>());
 
-  size_t count = 1000;
-  std::vector<std::unique_ptr<folly::ScopedEventBaseThread>> workers(count);
+  size_t count = 100;
```

Comment:
Nit: `constexpr` or `const` for both `size_t` vars.  I'd rename `count` to `connectionCount` or something like it to make things clearer.